protected-api-access-protected-api/README.md

```diff
@@ -0,0 +1,145 @@
+| **Supported account types**       | **Accounts in this organizational directory only (Single tenant)**           | Suggested value for this sample.                                                                   |
+| **Platform type**                 | **Web**                                                                      | Required value for this sample.                                                                    |
+| **Redirect URI**                  | `http://localhost:8080/redirect`                                             | Required value for this sample.                                                                    |
+| **Client secret**                 | _**Value** of the client secret (not its ID)_                                | :warning: Record this value immediately! <br/> It's shown only _once_ (when you create it).        |
+| **API Permissions**               | `user_impersonation`                                                         | Create a new delegated permission called user_impersonation.  Required value for this sample.      |
+
+
+### 3. Modify your existing Web API
+
+Update these settings to reference the new Web App
+
+| App registration <br/> setting    | Value for this sample app                                                    | Notes                                                                                              |
+|:---------------------------------:|:-----------------------------------------------------------------------------|:---------------------------------------------------------------------------------------------------|
+| **knownClientApplications**      | Client ID (UUID) of the application created in step 2.                       | Required value for this sample.                                                                    |
+
+
+> :information_source: **Bold text** in the tables above matches (or is similar to) a UI element in the Azure portal, while `code formatting` indicates a value you enter into a text box in the Azure portal.
+
+
+
+
+### 4. Update code sample with app registration values
+
+```javascript
+// 'Directory (tenant) ID' in Azure portal
+const tenant = ''
+
+const config = {
+  auth: {
+
+    // 'Application (client) ID' of app registration in Azure portal - this value is a GUID
+    clientId: '',
+
+    // Client secret 'Value' (not the ID) from 'Client secrets' in app registration in Azure portal
+    clientSecret: '',
+
+    // Full directory URL, in the form of https://login.microsoftonline.com/<tenant>
+    authority: `https://login.microsoftonline.com/${tenant}`
+  }
+}
+```
+
+
+### 3. Install package(s)
+
+To install Node.Js and MSAL libraries into your (virtual) environment:
+
+```bash
+npm install
+```
+
+## Run the application
+
+```bash
+node app.js
+```
+
+## Browse to the application
+
+Using postman or a similar app, navigate to **http://localhost:8080/me**
```

Comment:
What web app?  It's just an API, so it would need to be Postman or curl or something similar that can pass custom HTTP headers.  What will the tutorial ask the user to do?

Reply:
That question was added was before I looked at the code.

The title of the README says it's a **web app** and the app reg settings for a web app are specified. The app reg settings for the API say to use the registration for an **existing API**.

Thus, my confusion - everything indicated that this code sample at least _includes_ a web app because you're registering a web app.